docs/signed-apk-android.md

```diff
@@ -140,3 +138,7 @@ To enable Proguard, edit `android/app/build.gradle`:
  */
 def enableProguardInReleaseBuilds = true
 ```
+
+### Migrating old Android React Native apps to use App Signing by Google Play
```

Comment:
This seems like something we should mention in the release notes for the next release, right? cc @turnrye

Reply:
@TheSavior I think we should mention it -- thanks for bringing this to my attention!